recipes/NavierStokes/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "iscd-nstokes" %}
+{% set version = "0.0.0rc0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  # there are no tagged releases which so using archives of specific commits
+  url: https://github.com/ISCDtoolbox/NavierStokes/archive/2e14ef5fa4884674b7bbc2212af256652f02912c.zip
+  sha256: 7efe022f2b3ff88001b99c9cb83da5bdd15fb2b2084201662a55e4d666157116
+
+build:
+  number: 0
+  skip: true  # [py<3 or osx or win] # Cannot open include file: 'pthread.h': No such file or directory  
+  ignore_run_exports_from:
+    - {{ compiler('cxx') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - cmake
+    - make  # [not win]
+    - jom  # [win]    
+    - iscd-commons
+  run:
+    - iscd-commons
+
+test:
+  commands:
+    - test -f $PREFIX/bin/nstokes  # [unix]
```

Comment:
Please add a test for the shared libraries and headers. e.g.

```
- test -f $PREFIX/lib/libNavierStoves.so
- test -f $PREFIX/include/navier.h
```

Because most of your packages are not stable releases, these tests will help prevent releasing future versions of the packages that break their ABI/API.

Reply:
Thanks for taking the time to explain the test reason, I added them wherever I could